### DISCUSSION TITLE
Possible dead code in radio/src/gui/212x64/model_setup.cpp

### DISCUSSION THREAD

**Discussion by wingunder** [EdgeTX/edgetx#2121] [General]
Hi,
I was trying to compile an image with `GHOST` enabled and `CROSSFIRE` disabled and apart from some other problems, stumbled over the following code:
https://github.com/EdgeTX/edgetx/blob/29c0f2997f481d12969ef276680d2ddf1cb4c7f7/radio/src/gui/212x64/model_setup.cpp#L1043-L1068
To me, the code from line 1056 to 1067 seems to be unreachable, due to the `break` on line 1055.
Was this intentional, or is it a bug?
Regards

**Comment by raphaelcoeffic** [comment]
@wingunder that looks indeed like something that will never even compile in case `SPORT_MAX_BAUDRATE < 400000`.

**Reply by wingunder** [reply]
@raphaelcoeffic Indeed, when (`GHOST` _and_ `CROSSFIRE` are disabled) _and_ (`SPORT_MAX_BAUDRATE < 400000`), it won't compile. I guess it slipped through all the tests, as `GHOST` _and_ `CROSSFIRE` must also be disabled to hit this bug.

BTW, I have a fix for this, but I based it on your `no-multimodule` branch. Should I make a pull request for this?

**Reply by raphaelcoeffic** [reply]
That would be nice, yes.
